Integrations/integration-PostgreSQL.yml

```
@@ -4,8 +4,9 @@ commonfields:
-description: Query PostgreSQL Database
+description: Deprecated. Use Generic SQL
```

Small syntax update suggestion:
```suggestion
description: Deprecated. Use The Generic SQL integration instead.
```